packages/react/src/components/Authenticator/Authenticator.tsx

```diff
@@ -53,8 +53,11 @@ export function Authenticator({
   services,
   socialProviders,
   variation,
-}: AuthenticatorProps) {
-  const components = { ...defaultComponents, ...customComponents };
+}: AuthenticatorProps): JSX.Element {
+  const value = React.useMemo(
```

Comment:
Why do you think this should be in a useMemo?

Reply:
The `react/jsx-no-constructed-context-values` rule caught this. Longer explanation is that `components` was getting created with a new reference on each render, placing `components` inside the `useMemo`  wrapped `value` ensures  a static reference between renders